src/webhooks/pubsub/index.test.ts

```diff
@@ -37,8 +44,8 @@ describe('slack app', function () {
       headers: {
         authorization: 'Bearer 1234abcd',
       },
-    };
-    const reply = new MockReply();
+    } as FastifyRequest<{ Body: { message: { data: string } } }>;
+    const reply = new MockReply() as any;
     await pubSubHandler(request, reply);
     return reply;
```

Comment:
I think we also need to update the function definition to include a return type of `Promise<MockReply>` (since it is a subset of FastifyReply)

Reply:
I don't think we should be changing the function definition just for testing